core/src/main/java/com/linecorp/armeria/server/docs/DocServiceBuilder.java

```diff
@@ -241,6 +244,46 @@ public DocServiceBuilder exampleRequest(Iterable<?> exampleRequests) {
         return this;
     }
 
+    /**
+     * Adds a Javascript script to inject into the end of the debug page HTML. This can be used to customize
+     * the debug page (e.g., to provide a HeaderProvider for enabling authentication based on local storage).
+     * All scripts are concatenated into the content of a single script tag.
+     *
+     * <p>A common use case is to provide authentication of debug requests using a local storage access token,
+     * e.g., <pre>{@code
+     *   armeria.registerHeaderProvider(function() {
+     *     // Replace with fetching accesstoken using your favorite auth library.
+     *     return Promise.resolve({ authorization: 'accesstoken' });
+     *   });
+     * }</pre>
+     */
+    public DocServiceBuilder injectedScript(String... scripts) {
+        requireNonNull(scripts, "scripts");
+        return injectedScript(ImmutableList.copyOf(scripts));
+    }
+
+    /**
+     * Adds a Javascript script to inject into the end of the debug page HTML. This can be used to customize
```

Comment:
Aren't the scripts added to the `<head/>` section because you used `react-helmet`?

Reply:
Oops fixed